src/modules/esl-event-listener/core/api.ts

```diff
@@ -9,6 +11,10 @@ import type {
   ESLListenerDescriptorFn
```

Comment:
missed string import * from './types' as I understood

Reply:
Unfortunately export != import to current file even if we declare explicitly the export members

Reply:
Yep) Sorry for being inattentive